server/src/test/java/org/opensearch/search/approximate/ApproximatePointRangeQueryTests.java

```diff
@@ -86,301 +293,290 @@ public void testApproximateRangeWithDefaultSize() throws IOException {
+                        sort = new Sort(new SortField(numericType.fieldName, numericType.getSortFieldType(), true));
+                    }
+                    TopDocs topDocs = searcher.search(approximateQuery, size, sort);
+                    TopDocs topDocs1 = searcher.search(exactQuery, size, sort);
+                    // Verify we got the highest values first (DESC order)
+                    for (int i = 0; i < size; i++) {
+                        assertEquals("Mismatch at doc index " + i, topDocs.scoreDocs[i].doc, topDocs1.scoreDocs[i].doc);
                     }
-
                 }
             }
         }
     }
 
-    public void testApproximateRangeShortCircuit() throws IOException {
+    public void testApproximateRangeWithSizeDefault() throws IOException {
         try (Directory directory = newDirectory()) {
             try (RandomIndexWriter iw = new RandomIndexWriter(random(), directory, new WhitespaceAnalyzer())) {
                 int dims = 1;
-
-                long[] scratch = new long[dims];
                 int numPoints = 1000;
                 for (int i = 0; i < numPoints; i++) {
                     Document doc = new Document();
-                    for (int v = 0; v < dims; v++) {
-                        scratch[v] = i;
-                    }
-                    doc.add(new LongPoint("point", scratch));
+                    numericType.addField(doc, numericType.fieldName, i);
                     iw.addDocument(doc);
-                    if (i % 10 == 0) iw.flush();
                 }
                 iw.flush();
                 iw.forceMerge(1);
+                final int size = 10;
                 try (IndexReader reader = iw.getReader()) {
-                    try {
-                        long lower = 0;
-                        long upper = 100;
-                        final int size = 10;
-                        Query approximateQuery = new ApproximatePointRangeQuery(
-                            "point",
-                            pack(lower).bytes,
-                            pack(upper).bytes,
-                            dims,
-                            size,
-                            null,
-                            ApproximatePointRangeQuery.LONG_FORMAT
-                        );
-                        Query query = LongPoint.newRangeQuery("point", lower, upper);
-
-                        IndexSearcher searcher = new IndexSearcher(reader);
-                        TopDocs topDocs = searcher.search(approximateQuery, size);
-                        TopDocs topDocs1 = searcher.search(query, size);
-                        assertEquals(size, topDocs.scoreDocs.length);
-                        assertEquals(size, topDocs1.scoreDocs.length);
-                        assertEquals(topDocs1.totalHits.value(), 101);
-                    } catch (IOException e) {
-                        throw new RuntimeException(e);
-                    }
-
+                    long lower = 0;
```

Comment:
Would it be possible to generate random values that are within some range to be still valid ?